src/main/java/components/Task.java

```diff
@@ -0,0 +1,82 @@
+package components;
+
+/**
+ * Class for Task object encapsulate commonalities in subclasses.
+ */
+public class Task {
+    private int done;
+    private final String msg;
+
+    /**
+     * Constructor for a Task object.
+     * @param msg Description of the task.
+     */
+    public Task(String msg) {
+        this.msg = msg;
+        this.done = 0;
+    }
+
+    /**
+     * Get the msg of a Task.
```

Comment:
I think you missed the @return statement in the header, including some in the rest of the codebase.